server/src/matchRecipes.js

```diff
@@ -71,15 +87,15 @@ const resultToRecipe = (result: MatchedRecipeResult): Recipe => {
     ingredientsMatched,
     ingredientsNotMatched: R.difference(recipeIngredients, ingredientsMatched),
     nutritionalInfo: {
-      calories: 300,
-      fat: 10,
-      carbs: 30,
-      protein: 11,
-      sugar: 7
+      calories: result.calories,
+      fat: result.total_fat,
+      carbs: result.total_carbohydrates,
+      protein: result.protein,
+      sugar: result.sugars
     },
     imageUrl: "fake",
```

Comment:
Do you plan to get the image url in this PR or a future PR?

Reply:
I will probably do all the other stuff (number of reviews, "made it", prep_time in another batch. Maybe I will put ingredient category relationship there too.